vmdb/app/models/mixins/password_mixin.rb

```diff
@@ -13,6 +14,9 @@ def encrypt_column(column)
       #   password_encrypted  : Get the password in cryptext
       #   password_encrypted= : Set the password in cryptext
 
+      self.encrypted_columns ||= []
```

Comment:
Oh no wait you removed that... I like it better the other way where it initialized itself on first call